etl/package.json

```
@@ -1,71 +1,72 @@
-    },
-    "license": "CC0-1.0",
-    "author": "USEPA (https://www.epa.gov)",
-    "contributors": [
-      "Brad Cooper <brad.cooper@erg.com>",
-      "Caleb Schwind <caleb.schwind@erg.com",
-      "Max Diebold <max.diebold@erg.com>"
-    ],
-    "engines": {
-      "node": "18.x"
-    },
-    "repository": "github:USEPA/expert-query",
-    "scripts": {
-      "start": "npm run start_local_cmd",
-      "start_local_cmd": "cross-env NODE_ENV=local LOGGER_LEVEL=DEBUG nodemon --ignore 'app/tests' --inspect=9001 app/index.js",
-      "test": "echo \"Error: no test specified\" && exit 1"
-    },
-    "husky": {
-      "hooks": {
-        "pre-commit": "lint-staged"
-      }
-    },
-    "dependencies": {
-      "aws-sdk": "2.1232.0",
-      "canada": "0.1.0",
-      "change-case": "4.1.2",
-      "dotenv": "16.0.3",
-      "express": "4.18.2",
-      "fast-csv": "4.3.6",
-      "jsforce": "1.11.0",
-      "log4js": "6.7.0",
-      "node-cron": "3.0.2",
-      "numeral": "2.0.6",
-      "pg": "8.8.0",
-      "us-state-codes": "1.1.2"
-    },
-    "devDependencies": {
-      "browser-sync": "2.27.10",
-      "cross-env": "7.0.3",
-      "husky": "8.0.1",
-      "lint-staged": "13.0.3",
-      "nodemon": "2.0.20",
-      "prettier": "2.7.1"
-    },
-    "lint-staged": {
-      "linters": {
-        "app/**/*.{js,jsx,json,css}": [
-          "prettier --write",
-          "git add"
-        ]
-      },
-      "ignore": [
-        "app/css/epa/*.css",
-        "app/css/lib/*.css",
-        "app/js/lib/*.js"
-      ]
+  "name": "expert-query-etl",
+  "version": "1.0.0",
+  "description": "The ‘Expert Query’ ETL application is used to load data from ATTAINS (Oracle) into the ‘Expert Query’ Postgres DB.",
+  "keywords": [
+    "Expert Query",
+    "ATTAINS",
+    "ETL"
+  ],
+  "homepage": "https://github.com/Eastern-Research-Group/expert-query",
+  "bugs": {
+    "url": "https://github.com/Eastern-Research-Group/expert-query/issues"
+  },
+  "license": "CC0-1.0",
+  "author": "USEPA (https://www.epa.gov)",
+  "contributors": [
+    "Brad Cooper <brad.cooper@erg.com>",
+    "Caleb Schwind <caleb.schwind@erg.com",
+    "Max Diebold <max.diebold@erg.com>"
+  ],
+  "engines": {
+    "node": "18.x"
+  },
+  "repository": "github:USEPA/expert-query",
+  "scripts": {
+    "start": "npm run start_local_cmd",
+    "start_local_cmd": "cross-env nodemon -r dotenv/config --ignore 'app/tests' --inspect app/index.js",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "husky": {
+    "hooks": {
+      "pre-commit": "lint-staged"
     }
-  }
-  
\ No newline at end of file
+  },
+  "dependencies": {
+    "aws-sdk": "2.1232.0",
+    "canada": "0.1.0",
+    "change-case": "4.1.2",
+    "dotenv": "16.0.3",
+    "express": "4.18.2",
+    "fast-csv": "4.3.6",
+    "jsforce": "1.11.0",
+    "log4js": "6.7.0",
+    "node-cron": "3.0.2",
+    "numeral": "2.0.6",
+    "pg": "8.8.0",
+    "us-state-codes": "1.1.2"
+  },
```

```suggestion
  "dependencies": {
    "aws-sdk": "2.1232.0",
    "dotenv": "16.0.3",
    "express": "4.18.2",
    "log4js": "6.7.0",
    "node-cron": "3.0.2",
    "pg": "8.8.0"
  },
```

When I created this package.json file, I didn't spend very much time looking through the packages that were in there.

Packages to remove:
* canada - This is used fetching cities/provinces in Canada. We won't need this in the future.
* change-case - We probably won't need this. We can add this back in later if necessary.
* fast-csv - I think we will need a csv library, but I want to do some research as there may be a better library. This one doesn't seem to have much usage or support.
* jsforce - This is a Salesforce library, which we don't need.
* numeral - We probably won't need this. We can add this back in later if necessary.
* us-state-codes - We won't need this. If we do need to get state codes, we will just fetch them from attains (https://attains.epa.gov/attains-public/api/states) like we do in HMW.